features/authentication.feature

```diff
@@ -117,6 +117,25 @@ Feature: OAuth authentication
     Then the output should not contain "github.com password for mislav"
     And the file "../home/.config/hub" should contain "oauth_token: OTOKEN"
 
+  Scenario: Credentials from GITHUB_TOKEN
+    Given the GitHub API server:
+      """
+      post('/authorizations') {
+        assert_basic_auth 'mislav', 'kitty'
+        json :token => 'OTOKEN'
+      }
+      get('/user') {
+        json :login => 'mislav'
+      }
+      post('/user/repos') {
+        json :full_name => 'mislav/dotfiles'
+      }
+      """
+    Given $GITHUB_TOKEN is "feline0123"
+    When I successfully run `hub create`
+    Then the output should not contain "github.com password"
+    And the file "../home/.config/hub" should contain "oauth_token: feline0123"
```

Comment:
The token provided by environment variable should **never** result in `~/.config/hub` changing.